### DISCUSSION TITLE
Configure Playwright with synpress to handle metamask

### DISCUSSION THREAD

**Discussion by chaseFox0x** [Synthetixio/synpress#629] [Q&A] [answered]
Hello,

Has anyone implemented Synpress into Playwright? I have already done it using Cypress, but I cannot find any solutions to configure it in Playwright. I found this link: https://github.com/Synthetixio/synpress/issues/556 which means that there are plans to divide cypress and playwright solution - but it is a future.

**Comment by drptbl** [comment, marked as answer]
@chaseFox0x Playwright is now officially supported with synpress:
https://github.com/drptbl/synpress-examples
